CallsheetLocalizations/es.xcloc/Localized Contents/es.xliff

```diff
@@ -1552,7 +1552,7 @@
       </trans-unit>
       <trans-unit id="You may know them from" xml:space="preserve">
         <source>You may know them from</source>
-        <target state="translated">Los puedes conocer por</target>
+        <target state="translated">También conocido por</target>
```

Comment:
Yeah, I think @cliss uses this string in two semantically unrelated contexts, if I saw correctly. Once in the Persnickety Preferences for "show me more stuff with them in it", and once in the detail view for people, as in "You may know them from... Directing". I'd also prefer to have two separate strings for German, as then I could use a better translation there...

Reply:
This string is only used on the People view, for, in essence, "You know Stephen Spielberg as a director".

The two from Persnickety Preferences are:

* `Known For`
* `Show titles a person may be known from`

Reply:
Apologies, got it mixed up from memory! I actually meant the `Known For` string as the one that was reused.

I see the exact same string in the person view, as in `Known for... Acting` as in the Persnickety Preferences, as the Title for "show more known titles from this person".

So, in both places I have `Bekannt für` now, and I only see this string once. Dunno if it's clear what I mean :D

Reply:
To clarify: `Bekannt für` is the correct and very good translation in case of `They are known for acting`.

If I had a choice, though, I would use a different translation for the P.P. item: `Bekannt aus`, which sounds far more natural in that case.

Reply:
For the Spanish, I think you can try "you may know this person from", even if it's a little clunky

Reply:
I like that suggestion. Fixed it.

Reply:
Nice, sounds better in Spanish :)

Reply:
@DonSqueak https://github.com/cliss/callsheet-localizations/issues/55